traffic_ops/traffic_ops_golang/user/user.go

```diff
@@ -224,7 +225,7 @@ func (user *TOUser) Read() ([]interface{}, error, error, int) {
 		// role is a required field for the endpoint, but not for an item in the database
 		// I doubt a nil check is needed, but I'm including it just in case
 		if user.RoleName == nil {
-			return nil, nil, fmt.Errorf("role name is nil", err), http.StatusInternalServerError
+			return nil, nil, fmt.Errorf("role name is nil"), http.StatusInternalServerError
```

Comment:
nit: there's a couple instances of `fmt.Errorf` without any actual formatting and could just be `errors.New` instead